foyer/forcefield.py

```diff
@@ -118,6 +118,53 @@ def _topology_from_parmed(structure, non_element_types):
     positions = structure.positions
     return topology, positions
 
+
+def _topology_from_residue(res):
+    """Converts a openmm.app.Topology.Residue to openmm.app.Topology"""
+    topology = app.Topology()
+    chain = topology.addChain()
+    new_res = topology.addResidue(res.name, chain)
+
+    atoms = dict()  # omm.Atom in res : omm.Atom in topology
```

Comment:
```python
# {omm.Atom from res: omm.Atom in *new* topology}
```
Just to show dictionary relation a bit cleaner.
And this is the newly generated Topology correct?

Maybe add a bit more detail to comment, or a more detailed docstring. Not necessary in terms of PEP8 for private methods, but could be useful for debugging later.